src/components/com_tjcertificate/site/views/certificates/tmpl/my.php

```diff
@@ -105,10 +135,23 @@
 									?>
 								</td>
 								<td>
-									<?php echo $data[0]->title ? $data[0]->title : "-"; ?>
+									<?php
+										
+										if ($data[0]->title)
+										{
+											$title = $data[0]->title;
+										}
+										else
+										{
+											$title = $item->name;
+										}
+
```

Comment:
I think title will come in case of normal certificate you write the condition accordingly

Reply:
done